.github/workflows/scripts/create-framework-tag.sh

```
@@ -0,0 +1,19 @@
+TAG_NAME="framework/v${VERSION}"
+
+echo "🏷️ Creating framework tag: $TAG_NAME"
+
+# Create tag
+git tag "$TAG_NAME" -m "Release framework v${VERSION}"
+
+# Push tag
+git push origin "$TAG_NAME"
+
+echo "✅ Framework tag created and pushed: $TAG_NAME"
+echo "tag-created=true" >> $GITHUB_OUTPUT
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)